eng/Versions.props

```diff
@@ -50,7 +51,7 @@
     <MicrosoftCodeAnalysisCollectionsVersion>4.2.0-1.22102.8</MicrosoftCodeAnalysisCollectionsVersion>
     <MicrosoftDotNetXUnitExtensionsVersion>8.0.0-beta.24311.3</MicrosoftDotNetXUnitExtensionsVersion>
     <MicrosoftExtensionsDependencyModelVersion>7.0.0</MicrosoftExtensionsDependencyModelVersion>
-    <MicrosoftIORedistVersion>6.0.0</MicrosoftIORedistVersion>
+    <MicrosoftIORedistVersion>6.0.1</MicrosoftIORedistVersion>
```

Comment:
This required a coordinated change with VS; are you planning to drive that for 17.11?

Reply:
I will contact QB for VS and ask if the change is required for VS. It's possible we will not insert to VS. There could be a problem when we will have to backport something else that will be required to be inserted.

Reply:
As discussed offline, we will not update the MS.IO.Redist package version